### PR TITLE
Promote APIs in use by apps

### PR DESCRIPTION
**Pull request by NancyMcCallB**
Non-public APIs in use by apps promoted to public.

### CODE REVIEW

File: common/api/summary/core-react.exports.csv [hunk 1]
```diff
@@ -157,7 +157,7 @@ public;RadialMenuProps
 public;RadialSizeType = ProgressRadialProps["size"]
 public;RatioChangeResult
 public;ReactMessage
-internal;Rectangle 
+public;Rectangle
```
Comment (GerardasB) [inline common/api/summary/core-react.exports.csv:160]: Keep internal, use https://www.itwinjs.org/reference/core-geometry/cartesiangeometry/range2d/

File: common/api/summary/core-react.exports.csv [hunk 2]
```diff
@@ -174,7 +174,7 @@ public;SettingsProvidersChangedEventArgs
 public;SettingsTabEntry
 public;SettingsTabsProvider
 internal;shallowDiffers: (a:
-internal;Size 
+public;Size
```
Comment (GerardasB) [inline common/api/summary/core-react.exports.csv:177]: Keep internal, use inline object or https://www.itwinjs.org/reference/core-geometry/cartesiangeometry/xy/ for comparison.

File: common/api/summary/core-react.exports.csv [hunk 3]
```diff
@@ -40,7 +40,7 @@ public;ContextMenuItemProps
 public;ContextMenuProps 
 public;ContextSubMenu 
 public;ContextSubMenuProps 
-internal;Corner
+public;Corner
```
Comment (GerardasB) [inline common/api/summary/core-react.exports.csv:43]: Keep internal, this is only used with `Rectangle`.

**Review by GerardasB — CHANGES_REQUESTED** [review]

File: common/api/summary/core-react.exports.csv [hunk 4]
```diff
@@ -140,7 +140,7 @@ public;OmitChildrenProp
 public;Orientation
 public;OutsideClickEvent = PointerEvent | MouseEvent | TouchEvent
 internal;percentInRange(percent: number): number
-internal;Point 
+public;Point
```
[the remark this answers is not in the document]
Comment (raplemie) [reply common/api/summary/core-react.exports.csv:143]: If we can't remove it right away, maybe we can document and point users to a public version of these ?

File: ui/appui-react/src/appui-react/UiFramework.ts [hunk 5]
```diff
@@ -614,15 +614,15 @@ export class UiFramework {
     UiFramework.dispatchActionToStore(ConfigurableUiActionId.UseToolAsToolSettingsLabel, value, true);
   }
 
-  /** @alpha */
+  /** @public */
```
Comment (raplemie) [inline ui/appui-react/src/appui-react/UiFramework.ts:617]: Just so user dont have to search the setter method to understand the difference.
```suggestion
  /** When `true`, panels will close as soon as the mouse leave the panel.
   * When `false` (default), panels will close on next click outside the panel.
   * @public */
```

File: ui/appui-react/src/appui-react/configurableui/InternalConfigurableUiManager.ts [hunk 6]
```diff
@@ -26,26 +26,26 @@ import { InternalModalDialogManager } from "../dialog/InternalModalDialogManager
 import { SyncUiEventDispatcher } from "../syncui/SyncUiEventDispatcher";
 
 /** Ui Activity Event Args interface.
- * @internal
+ * @public
```
Comment (raplemie) [inline ui/appui-react/src/appui-react/configurableui/InternalConfigurableUiManager.ts:29]: This file is not exported in the barrel, so even if they are marked public, they will not show up.
These events are part of the internal timers and the results of the timers are exposed in events like "onFronstageDeactivatedEvent". Do we really want to expose those ? (If so, we should move these events somewhere else as well).

Comment (NancyMcCallB) [reply ui/appui-react/src/appui-react/configurableui/InternalConfigurableUiManager.ts:29]: DR uses this in their testing. I'll check in with them.

File: common/api/summary/core-react.exports.csv (hunk 4 above)
[the remark this answers is not in the document]
Comment (GerardasB) [reply common/api/summary/core-react.exports.csv:143]: There is no public version, it's just utility classes for working with PointProps structure. Apps can use their own util libraries or use ones provided by core-geometry.

File: ui/appui-react/src/appui-react/widgets/WidgetDef.tsx [hunk 7]
```diff
@@ -405,14 +405,14 @@ export class WidgetDef {
...
-   * @alpha
+   * @public
    */
   public show() {
     InternalFrontstageManager.onWidgetShowEvent.emit({ widgetDef: this });
   }
 
   /** Opens the widget and expands it to fill full size of the stage panel.
-   * @alpha
+   * @public
```
[the remark this answers is not in the document]
Comment (NancyMcCallB) [reply ui/appui-react/src/appui-react/widgets/WidgetDef.tsx:415]: Apps use them. You want to leave them alpha? There's not another way to do what they want to do.

File: common/api/summary/core-react.exports.csv (hunk 4 above)
[the remark this answers is not in the document]
Comment (NancyMcCallB) [reply common/api/summary/core-react.exports.csv]: Jason said he will refactor to use PointProps, so I've moved it back to internal.

**Review by raplemie — APPROVED** [review]

**Review by GerardasB — APPROVED** [review]

**Pull request merged**